CHANGELOG.md

```diff
@@ -39,6 +40,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
   in trait implementations where methods cannot fail.
 - A new [process](https://github.com/rust-embedded/embedded-hal#how-to-add-a-new-trait)
   has been adopted for the addition of traits to the embedded-hal.
+- The ADC `Channel` trait now uses a constant to represent the IDs.
 - The minimum supported Rust version is 1.35 due to [this issue](https://github.com/rust-lang/rust/issues/54973).
```

Comment:
This can now also be removed, since the associated feature isn't used anymore, right?
```suggestion

```

Reply:
The `-alpha.1` version would still need 1.35 but after merging this PR we could decrease the MSRV again for the next release.